### PR TITLE
add support for >,>=,<,<=

### PR DESCRIPTION
**Pull request by collinglass**
This commit adds support for doing less than, less than or equal to, greater than, and greater than or equal to. I added tests as well.

Please give feedback if not merged.

### CODE REVIEW

File: expr.go [hunk 1]
```diff
@@ -132,6 +132,86 @@ func (neq NotEq) ToSql() (sql string, args []interface{}, err error) {
...
+// Lt is syntactic sugar for use with Where/Having/Set methods.
+// Ex:
+//     .Where(Lt{"id": 1})
+type Lt map[string]interface{}
+
+func (lt Lt) toSql(opposite, orEq bool) (sql string, args []interface{}, err error) {
+	var (
+		exprs   []string
+		opr     string = "<"
+		nullOpr string = "IS"
```
Comment (lann) [inline expr.go]: I know it mirrors my implementation of Eq, but I think it's too confusing to treat nil/null this way for comparisons.

Comment (collinglass) [inline expr.go]: Understandable, any suggestions?

It could throw an error like is the case for an array or slice. What do you think?

**Comment by lann** [comment]
Unfortunately I will not be able to review or merge major changes for the forseeable future; see README.

**Comment by technosophos** [comment]
Lann has moved Squirrel to the Masterminds project, and is still the architect, but @mattfarina and I will be helping him maintain. We'll start reviewing pull requests and see if we can get this in.

**Comment by mattfarina** [comment]
@collinglass there was a bug causing tests to fail. Can you merge in the latest or rebase your changes on top of the latest so we can see it with passing tests?

**Comment by collinglass** [comment]
sure

**Comment by collinglass** [comment]
merged

File: expr.go (hunk 1 above)
Comment (technosophos) [inline expr.go]: If there are no other suggestions, I recommend merging as-is.

Comment (lann) [inline expr.go]: I think an error would be appropriate.

**Comment by collinglass** [comment]
Just updated to throw error on null value

**Comment by collinglass** [comment]
Hi! What's the status on this?

**Comment by technosophos** [comment]
I think we can probably merge this now. Any objections @mattfarina ?

**Comment by mattfarina** [comment]
@technosophos have you given this a good look over? I've not but it seems sane on the surface.

**Comment by technosophos** [comment]
I have. It's mostly sugar for various comparison operations. @lann looked at this a long time ago, and his only complaint was the nil treatment, which has since been removed. The style seems to me to be in keeping with Squirrel's original intent. So I am good with it unless Lann jumps back in and says otherwise.

**Pull request merged**